src/System.Net.Security/ref/System.Net.Security.csproj

```diff
@@ -2,10 +2,29 @@
   <PropertyGroup>
     <ProjectGuid>{A7488FC0-9A8F-4EF9-BC3E-C5EBA47E13F8}</ProjectGuid>
     <Configurations>netcoreapp-Debug;netcoreapp-Release;uap-Debug;uap-Release</Configurations>
+    <!-- This is needed so that code for TlsCipherSuite will have no namespace (causes compile errors) when used within T4 template  -->
+    <DefineConstants>PRODUCT</DefineConstants>
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System.Net.Security.cs" />
   </ItemGroup>
+  <!-- This file depends on IANA registry. We do not want anyone's build to break after the update-->
+  <!-- or if they don't have internet connection -->
```

Comment:
More than that, doing a build shouldn't be making internet connections and downloading data unless someone actually opts in to that.